### DISCUSSION TITLE
Concept: DnD between RGL

### DISCUSSION THREAD

**Discussion by avasuro** [react-grid-layout/react-grid-layout#1872] [Show and tell]
Hi,

I would like to try to implement the ability to drag-and-drop grid layout items between different instances of the RGL.

I saw several issues here with similar request:
https://github.com/react-grid-layout/react-grid-layout/issues/688
https://github.com/react-grid-layout/react-grid-layout/issues/595 
https://github.com/react-grid-layout/react-grid-layout/issues/586 
https://github.com/react-grid-layout/react-grid-layout/issues/459 
https://github.com/react-grid-layout/react-grid-layout/issues/11 

Also, there is one PR that implemented this functionality (but it is outdated):
https://github.com/react-grid-layout/react-grid-layout/pull/684 

As implementation of this feature will involve a lot of my time, before I would do that I would like to get a feedback from this lib devs:
- Just in case: Is this project still alive and is going to be developed in near future?
- Do you need such functionality at all (wheter there is a chance that this functionality will be merged if implemented)?
- Would you be ready to look into this tread from time to time to give a feedback on something that I'm going to post?

### Some information to start

I already made an example of such implementation that works "on top" of current RGL implementation, and uses browser native DnD flow. This implementation is full of hacks, but they would be removed if this implementation become a part of this library (as they are needed only for one purpose - to not make changes in internal code of this library). Also, there are still some bugs that could be fixed only by modification of RGL internal code.

Because demo implementation that I've made uses native browser DnD - it potentially allows grid item to be dropped everywhere, even between iframes/browser windows and even to the other applications (which could be useful for native apps developed using Electron or similar).

I'll show an example after I get some feedback for this message ;)

Waiting for your feedback.

Regards,
Alex

**Comment by avasuro** [comment]
@STRML Not sure what is the usual process, but I saw that in [this post](https://github.com/react-grid-layout/react-grid-layout/discussions/1884) you answered after you've been tagged, so tagging you here too. Sorry for disturb :)

**Comment by taziksh** [comment]
@avasuro did you end up working on this?

**Reply by avasuro** [reply]
No, I didn't received any answer from project maintainer, so big chances that this lib is dead (or maintainer is not interested in such feature).
